lib/api/element-commands/getComputedLabel.js

```diff
@@ -0,0 +1,61 @@
+const BaseElementCommand = require('./_baseElementCommand.js');
+
+/**
+ * Returns the computed WAI-ARIA label of an element.
+ *
+ * @example
+ * module.exports = {
+ *   demoTest(browser) {
+ *     browser.getComputedLabel('#main ul li a.first', function(result) {
```

Comment:
this should be name as `getAccessibleName`, as per [selenium implementation](https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/lib/webdriver.js#L2644); also same for getAriaRole() instead of getComputedRole

also, getAccessibleName() for

Reply:
okay, I named them `getComputedRole` and `getComputedLabel` as it was mentioned in the endpoints section of https://w3c.github.io/webdriver/#endpoints.

![image](https://user-images.githubusercontent.com/8705386/123906922-aadd1400-d992-11eb-9252-d773ccf4ee20.png)

Should we still change the name according to selenium implementations?

Reply:
We should go with the getAriaRole and getAccessibleName because that's part of the selenium bindings api and will be more familiar to users.